modules/cms/classes/CodeBase.php

```diff
@@ -155,6 +155,14 @@ public function __set($name, $value)
      */
     public function __isset($name)
     {
-        return isset($this->page->{$name});
+        if (isset($this->page->components[$name]) || isset($this->layout->components[$name])) {
+            return true;
+        }
+
+        if (isset($this->page->{$name})) {
+            return $value;
```

Comment:
@RickAcb I presume this should be returning `true` here, as `$value` is undefined.
```suggestion
            return true;
```